### DISCUSSION TITLE
Is there a way to return the site collection admin list for a Microsoft 365 Group?

### DISCUSSION THREAD

**Discussion by plamber** [pnp/pnpcore#1308] [General]
Hello,

When using the 'GetSiteCollectionAdminsAsync' method, it retrieves the list of site collection admins in two different ways: for non-groupified sites, it returns the site collection admins directly, whereas for groupified sites, it returns the owners of the associated owners group of the site.

In certain scenarios, it may be necessary to treat groupified sites in the same manner as non-groupified sites, returning the list of "real" site collection admin objects. Is there an alternative solution to address this requirement, or could a feature be added to 'GetSiteCollectionAdminsAsync' that allows for a flag to bypass the current group logic?

Your input is greatly appreciated.

Best regards,
Patrick

**Comment by jansenbe** [comment]
By default there's only a claim in the site collection admins pointing to the Microsoft 365 Group owners, hence we're returning that. Wondering why you would have additional site collection admins for these sites?

**Comment by plamber** [comment]
Hi,
we come across a couple of times with the scenario that IT Administrators want to assign a couple of security groups as site collection owners to a Microsoft 365 Group to "simplify support" requests. 

To achieve this goal you need to have the possibility to treat the Groupified SharePoint site as a "normal" site.

The alternative is always to perform direct calls by your own. I thought, however, that it could be useful add a flag to cover such scenarios directly in the SDK.

Cheers
